src/components/NFTPage/EventList.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <DropDownList
+    class="w-full mb-[250px]"
+    title="Item Activity"
+  >
+    <template #titleIcon>
+      <IconActivity />
+    </template>
+    <template #content>
+      <table v-if="nftHistory.length" class="w-full table-fixed text-[10px]">
+        <thead class="border-b-shade-gray border-b-[2px]">
+          <tr class="text-medium-gray py-[12px]">
+            <th><Label :text="$t('nft_details_page_activity_list_event')" /></th>
+            <th><Label class="break-normal" :text="$t('nft_details_page_activity_list_event_price')" /></th>
+            <th><Label :text="$t('nft_details_page_activity_list_event_from')" /></th>
+            <th><Label :text="$t('nft_details_page_activity_list_event_to')" /></th>
+            <th><Label :text="$t('nft_details_page_activity_list_event_date')" /></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr
+            v-for="event in nftHistory"
+            :key="`${event.txHash}event`"
+            class="py-[12px] border-b-shade-gray border-b-[1px]"
+          >
+            <td>
+              <Label :text="$t('nft_details_page_activity_list_event_collect')">
+                <template #prepend>
+                  <IconCircle />
+                </template>
+              </Label>
+            </td>
+            <td>
+              <Label>{{ event.price }}</Label>
+            </td>
+            <td><Label text="-" /></td>
+            <td>
+              <LinkV2 :to="`/${event.toWallet}`">
+                <Label class="break-all">{{
+                  displayNameList[event.toWallet] | ellipsis
```

Comment:
Suggest to create a computed property e.g. `populatedEvents` that loop through event array by `array.map` and add a `displayName` key to event array item then return the populated event list, so that no need to pass displayNameList as props